README
NAME
    Dist::Zilla::PluginBundle::BINGOS - BeLike::BINGOS when you build your
    dists

VERSION
    version 0.18

SYNOPSIS
       # in dist.ini
       [@BINGOS]

DESCRIPTION
    This is a Dist::Zilla PluginBundle. It is roughly equivalent to the
    following dist.ini:

      [@Basic]

      [MetaJSON]
      [PodSyntaxTests]
      [PodCoverageTests]

      [PodWeaver]
      [PkgVersion]
      [GithubMeta]

      [ChangelogFromGit]
      file_name = Changes
      tag_regexp = ^(\\d+\\.\\d+)$
      max_age = 1825

      [ReadmeFromPod]
      [Test::Compile]

      [Clean]

    This PluginBundle also supports PluginRemover, so dropping a plugin is
    as easy as this:

       [@BINGOS]
       -remove = PluginIDontWant

  METHODS
    "configure"
        See Dist::Zilla::PluginBundle::Role::Easy.

AUTHOR
    Chris Williams <chris@bingosnet.co.uk>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2013 by Chris Williams.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.



Changes
==================================================
Changes from 2008-08-02 00:00:00 +0000 to present.
==================================================

-----------------------------------------
version 0.18 at 2013-08-01 09:02:03 +0000
-----------------------------------------

  Change: 9fa28fd8d7f17d24ef10f34918d48e0bede67804
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2013-08-01 10:02:03 +0000

    Add 'Dist::Zilla::Role::PluginBundle::PluginRemover' support 

-----------------------------------------
version 0.16 at 2013-01-10 15:12:36 +0000
-----------------------------------------

  Change: 8cd0bc661aa11e8a5b89857dbacc381208400ed2
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2013-01-10 15:12:36 +0000

    ChangelogFromGit's behaviour has changed, update to support 

-----------------------------------------
version 0.14 at 2011-09-19 13:58:17 +0000
-----------------------------------------

  Change: fd41f1fe5d1c4e39219cee7855a2e5eb970d5af7
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2011-09-19 14:58:17 +0000

    Switch CompileTests to Test::Compile 

-----------------------------------------
version 0.12 at 2011-08-05 17:20:45 +0000
-----------------------------------------

  Change: 1d5aa1c8c1d79538d428ff2d081091faa8c7e40d
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2011-08-05 18:20:45 +0000

    [rt.cpan.org #70053] DZP::ChangelogFromGit not listed as a dependency

    reported by Dave Golden 

-----------------------------------------
version 0.10 at 2011-05-03 12:45:28 +0000
-----------------------------------------

  Change: 656a38dfe915f3a08edf0752bb8bfce9e3a0f70d
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2011-05-03 13:45:28 +0000

    Update prereqs. Thanks rjbs for spotting a missing one 

-----------------------------------------
version 0.08 at 2011-04-29 16:40:17 +0000
-----------------------------------------

  Change: 34c1b1164519f346a12444d4039ebaa65c275d65
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2011-04-29 17:40:17 +0000

    Add ::Clean plugin for auto-cleaning after 'release' 

-----------------------------------------
version 0.06 at 2011-02-22 23:13:37 +0000
-----------------------------------------

  Change: cd07fa8abd3dc131abd3201603c46ab0fcd47d09
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2011-02-22 23:13:37 +0000

    Fix mistake in POD, spotted by Glenn Fowler (CEBJYRE) 

-----------------------------------------
version 0.04 at 2010-11-23 00:14:58 +0000
-----------------------------------------

  Change: 4e1e013e9cc91c555365f05db5158ecab0380f61
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2010-11-23 00:14:58 +0000

    Bump max_age for ChangesFromGit to 1825 ( 5 years ) 

-----------------------------------------
version 0.02 at 2010-06-17 21:16:42 +0000
-----------------------------------------

  Change: a8734d21481dcbb5d72e12369b5660ef0b04e51c
  Author: Chris 'BinGOs' Williams <chris@bingosnet.co.uk>
  Date : 2010-06-17 22:16:42 +0000

    Why, hello. 

================
End of releases.
================


Makefile.PL

use strict;
use warnings;

use 5.008005;

use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  "ABSTRACT" => "BeLike::BINGOS when you build your dists",
  "AUTHOR" => "Chris Williams <chris\@bingosnet.co.uk>",
  "BUILD_REQUIRES" => {},
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => "6.30"
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-BINGOS",
  "EXE_FILES" => [],
  "LICENSE" => "perl",
  "NAME" => "Dist::Zilla::PluginBundle::BINGOS",
  "PREREQ_PM" => {
    "Dist::Zilla" => "4.200006",
    "Dist::Zilla::Plugin::ChangelogFromGit" => "0.006",
    "Dist::Zilla::Plugin::Clean" => "0.02",
    "Dist::Zilla::Plugin::GithubMeta" => "0.10",
    "Dist::Zilla::Plugin::MetaJSON" => "4.200006",
    "Dist::Zilla::Plugin::PkgVersion" => "4.200006",
    "Dist::Zilla::Plugin::PodCoverageTests" => "4.200006",
    "Dist::Zilla::Plugin::PodSyntaxTests" => "4.200006",
    "Dist::Zilla::Plugin::PodWeaver" => "3.101641",
    "Dist::Zilla::Plugin::ReadmeFromPod" => "0.14",
    "Dist::Zilla::Plugin::Test::Compile" => "1.112400",
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => "0.102",
    "Moose" => "1.08"
  },
  "TEST_REQUIRES" => {
    "Capture::Tiny" => 0,
    "Test::More" => "0.88",
    "blib" => 0
  },
  "VERSION" => "0.18",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  my $tr = delete $WriteMakefileArgs{TEST_REQUIRES};
  my $br = $WriteMakefileArgs{BUILD_REQUIRES};
  for my $mod ( keys %$tr ) {
    if ( exists $br->{$mod} ) {
      $br->{$mod} = $tr->{$mod} if $tr->{$mod} > $br->{$mod};
    }
    else {
      $br->{$mod} = $tr->{$mod};
    }
  }
}

unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



